practice-typescript/src/scripts/services/user.ts

```diff
@@ -9,13 +9,18 @@ class ApiService {
     this.resourceUrl = `${BASE_URL}/${USERS}`;
   }
 
-  async getAll(): Promise<UserModel[]> {
+  async getAll(): Promise<UserModel[] | null> {
```

Comment:
Why null here? If you don't have any users. Your should return an empty array

Reply:
fixed